### PR TITLE
Update demystifying python docsv2

### PR DESCRIPTION
**Pull request by drelliche**

### CODE REVIEW

File: demystifying_python/demystifying_python.md [hunk 1]
```diff
@@ -2,14 +2,21 @@
...
 author:   Meredith Lee
 email:    leemc@chop.edu
-version: 1.2.2
-module_template_version: 3.0.0
+version: 1.2.3
+current_version_description: Updated highlight boxes and metadata
+module_type: standard
+docs_version: 2.0.0
 language: en
 narrator: UK English Female
```
Comment (rosemm) [inline demystifying_python/demystifying_python.md:10]:
```suggestion
narrator: UK English Female
mode: Textbook
```

File: demystifying_python/demystifying_python.md [hunk 2]
```diff
@@ -21,11 +28,30 @@ After completion of this module, learners will be able to:
 
 @end
 
-link:  https://chop-dbhi-arcus-education-website-assets.s3.amazonaws.com/css/styles.css
+good_first_module:
```
Comment (rosemm) [inline demystifying_python/demystifying_python.md:31]:
```suggestion
good_first_module: false
```

File: demystifying_python/demystifying_python.md [hunk 3]
```diff
@@ -21,11 +28,30 @@ After completion of this module, learners will be able to:
 
 @end
 
-link:  https://chop-dbhi-arcus-education-website-assets.s3.amazonaws.com/css/styles.css
+good_first_module: 
+coding_required:
```
Comment (rosemm) [inline demystifying_python/demystifying_python.md:32]:
```suggestion
coding_required: false
```

Right? I didn't actually verify this in the module

File: demystifying_python/demystifying_python.md [hunk 4]
```diff
@@ -21,11 +28,30 @@ After completion of this module, learners will be able to:
...
+- [1.0.2](https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/education_modules/466799a081d2cb74d155dc0a26951d3492b81f8c/demystifying_python/demystifying_python.md): Initial Version.
+
+
+@end
+
+import: https://raw.githubusercontent.com/arcus/education_modules/main/_module_templates/macros.md
 
-script:  https://code.jquery.com/jquery-3.6.0.slim.min.js
 
 @gifPreload
```
Comment (rosemm) [inline demystifying_python/demystifying_python.md:56]: We should delete this gifPreload macro, too. It's in the main macros file.

File: demystifying_python/demystifying_python.md (hunk 3 above)
Comment (drelliche) [reply demystifying_python/demystifying_python.md:32]: Oh I totally left these blank with the intention of skimming through the module to evaluate and then forgot! Doh!

File: demystifying_python/demystifying_python.md (hunk 4 above)
Comment (drelliche) [reply demystifying_python/demystifying_python.md]: Good call, done!

**Review by rosemm — APPROVED** [review]

**Pull request merged**